src/components/Task.vue

```diff
@@ -107,7 +108,7 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 				</form>
 			</div>
 			<ol v-if="!task.hideSubtasks" :calendarID="task.calendar.uri">
-				<TaskBodyComponent v-for="subtask in task.subTasks"
+				<TaskBodyComponent v-for="subtask in sort(task.subTasks, sortOrder, sortDirection)"
```

Comment:
For some reason this line gives me an error:
<img width="1262" alt="warn" src="https://user-images.githubusercontent.com/2496460/50057487-4d35a300-016b-11e9-82db-8ff1483f387d.png">

Reply:
I made this change https://github.com/nextcloud/tasks/pull/223/commits/04b1bdeaed232a66a8cd4d864901e01e778caa12
according to Evan's comment: https://github.com/vuejs/vue/issues/3276#issuecomment-232818665

Reply:
Interesting, I didn’t know that.